### DISCUSSION TITLE
run logs are not available while it is running

### DISCUSSION THREAD

**Discussion by jw-lilly** [community/community#48806] [Actions] [answered]
### Select Topic Area

Product Feedback

### Body

I've just noticed that I have limited access to logs for a run while it is still running.  The live view is massively helpful, but it is only available in the UI, and it only provides logs that were generated since I opened the UI.  

Is there little demand for the current full log from a run log while it's still running?  I'm thinking particularly of longer running tasks.  Of course there are workarounds, but I just wanted to see if this is a hot topic or not.

When I use the CLI, I get this:
```
job 11659541058 is still in progress; logs will be available when it is complete
```

Upvote if you feel the same, comment if you're aware of a GitHub position on this, 👎 or ignore if you don't care.  Thanks!

**Comment by sayhiben** [comment, marked as answer]
Agree - there are several unactioned threads on this topic over the past few years, e.g., https://github.com/orgs/community/discussions/35363

**Reply by jw-lilly** [reply]
Thanks @sayhiben .  I'll be waiting not-so-patiently at  #44250  and https://github.com/actions/runner/issues/886#issuecomment-1405845909
